calcom/cal.com
# fix: username + replaced with -

## What does this PR do?

replaces all '+' with '-' in the username string.

Fixes # [(issue)](https://github.com/calcom/cal.com/issues/10078)




## Type of change




## How should this be tested?


![image](https://github.com/calcom/cal.com/assets/33975149/5ec1c5fd-ecea-4605-8d7f-d3b16a01b52f)

Did console.log to check the new string in the browser console. Got the desired log in the browser. Could not test the functionality e2e in local because sign up functionality doesn't work in local therefore used console.log to check the final object.

![image](https://github.com/calcom/cal.com/assets/33975149/ded3cb8d-92dc-414b-8b9c-f8a490dbda8d)

Inputs used as username to test => ayush+gov, ayush-gov+,  +ayush-gov+, ++a 


## Mandatory Tasks
